compiler/rustc_target/src/spec/targets/armv7a_vex_v5.rs

```diff
@@ -0,0 +1,37 @@
+use crate::spec::{Cc, LinkerFlavor, Lld, PanicStrategy, RelocModel, Target, TargetOptions};
+
+const LINK_SCRIPT: &str = include_str!("./armv7a_vex_v5_linker_script.ld");
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "armv7a-none-eabihf".into(),
+        metadata: crate::spec::TargetMetadata {
+            description: Some("Armv7-A Cortex-A9 VEX V5 Brain, VEXos".into()),
+            tier: Some(3),
+            host_tools: Some(false),
+            std: Some(true),
+        },
+        pointer_width: 32,
+        data_layout: "e-m:e-p:32:32-Fi8-i64:64-v128:64:128-a:0:32-n32-S64".into(),
+        arch: "arm".into(),
+        options: TargetOptions {
+            os: "vexos".into(),
+            vendor: "vex".into(),
+            exe_suffix: ".elf".into(),
```

Comment:
I still think we should get rid of this suffix, because

1. It's not how vexide builds currently behave and will therefore break `cargo v5` and possibly other tooling.
2. Users don't typically expect ELF executables to have a file extension.
3. It's inconsistent with (afaict) all other Rust targets.